lib/vquic/quiche.c

```diff
@@ -808,7 +849,12 @@ static CURLcode http_request(struct connectdata *conn, const void *mem,
   if(stream3_id < 0) {
     H3BUGF(infof(data, "quiche_h3_send_request returned %d\n",
                  stream3_id));
-    result = CURLE_SEND_ERROR;
+    if(stream3_id == QUICHE_H3_ERR_STREAM_BLOCKED) {
+      result = CURLE_AGAIN;
```

Comment:
I realize this is pretty much the only way we can deal with this now, but getting stream blocked here and just blindly retry it at the next possible moment is going to cause some crazy busy-looping. I would imagine that this rather needs a more clever back-off mechanism or a timer to make the retry hold off a bit.